components/Vote/Election.js

```diff
@@ -22,12 +27,11 @@ const styles = {
   wrapper: css({
     width: '100%',
     position: 'relative',
-    minHeight: 200,
-    marginTop: 20
+    minHeight: 200
   }),
   header: css({
     position: 'sticky',
-    padding: '10px 0',
+    padding: '20px 0',
     top: HEADER_HEIGHT,
```

Comment:
subtract 1 pixel here
<img width="861" alt="screen shot 2018-10-15 at 14 30 29" src="https://user-images.githubusercontent.com/410211/46952741-099bbb80-d08c-11e8-861c-752165fb6ec5.png">
I had to do the same for the hr lines in [the header](https://github.com/orbiting/republik-frontend/blob/7ce7e0fd576d15a8f6bfa3d9e7f4d6ac86c91025/components/Frame/Header.js#L172)

I don't recall exactly why, something about how the multiple elements are pushed.